crates/db-address-space/src/address_space.rs

```diff
@@ -164,6 +169,13 @@ mod hotplug {
     }
 
     impl AddressSpaceAtomic {
+        /// Convert a [GuestMemoryMmap] object into `GuestMemoryAtomic<GuestMemoryMmap>`.
+        pub fn convert_into_vm_as(
+            gm: GuestMemoryMmap,
```

Comment:
Is it possible to change it in the previous patch which introduced it?

Reply:
Hope to keep the refactor history:)